diff --git a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
index af2a39a..04eea16 100644
--- a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
+++ b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
@@ -12,10 +12,15 @@ exports.LoadUtils = () => {
     window.WWebJS.sendSeen = async (chatId) => {
         const chat = await window.WWebJS.getChat(chatId, { getAsModel: false });
         if (chat) {
-            window.Store.WAWebStreamModel.Stream.markAvailable();
-            await window.Store.SendSeen.sendSeen(chat);
-            window.Store.WAWebStreamModel.Stream.markUnavailable();
-            return true;
+            try {
+                window.Store.WAWebStreamModel.Stream.markAvailable();
+                await window.Store.SendSeen.sendSeen(chat);
+                window.Store.WAWebStreamModel.Stream.markUnavailable();
+                return true;
+            } catch (error) {
+                console.error("Suppressed sendSeen error:", error);
+                return false;
+            }
         }
         return false;
     };
